packages/vue-components/src/Popover.vue

```diff
@@ -21,7 +21,6 @@
       :hide-triggers="triggers"
       :placement="placement"
       :delay="0"
-      popper-class="v-popper__popper--skip-transition"
```

Comment:
👀 This does the trick for the previous issues, seems like everything was related. 

(just speculation: perhaps `v-popper__popper--skip-transition` isn't meant to be used externally? -- not sure)

Reply:
Think we could @ryoarmanda to do another check too, since a lot of the issues were device specific.

Reply:
> 👀 This does the trick for the previous issues, seems like everything was related.
> 
> (just speculation: perhaps `v-popper__popper--skip-transition` isn't meant to be used externally? -- not sure)

Hmm I believe so! floating-vue's [docs mention `v-popper__popper--skip-transition`](https://floating-vue.starpad.dev/guide/css.html) as a possible helper class though it seems like it's best used in conjunction with floating-vue's CSS themes as a dynamic class.

The hover issues were fixed in [`55a888`](https://github.com/MarkBind/markbind/pull/1868/commits/55a888916273b5c30f92acc5417a008a5c27cd57)—had to separate the handling of `<input>`s because wrapping `<input>` in a wrapper element results in floating-vue being unable to detect its focus events, but failing to wrap other tags in a wrapper element prevents floating-vue from detecting mouse events (which was what caused the un-openable tooltip in index.md).

Reply:
The behaviours are now fixed on my end :+1: